apps/council-ui/pages/index.tsx

```diff
@@ -104,44 +107,43 @@ export default function Index(): ReactElement {
         </div>
       </div>
 
-      <div className="w-full p-24 bg-base-200">
+      <div className="w-full md:p-24 p-8 bg-base-200">
         <div className="max-w-5xl m-auto">
           <div className="flex flex-col items-center gap-8">
-            <div className="text-4xl font-medium">Council 🤝 DAOs </div>
+            <h2 className="text-4xl font-medium">Council 🤝 DAOs </h2>
 
             <div className="flex flex-wrap justify-center gap-6">
               <div className="w-48 daisy-card bg-neutral text-neutral-content">
                 <div className="items-center text-center daisy-card-body">
                   <h2 className="daisy-card-title">Yearn</h2>
-                  {/* <Image src={} /> */}
-                  <img
+                  <Image
                     alt="yearn logo"
                     height={80}
                     width={80}
-                    src="https://cryptologos.cc/logos/yearn-finance-yfi-logo.png"
+                    src="/images/yearn-finance-yfi-logo.png"
                   />
                 </div>
               </div>
               <div className="w-48 daisy-card bg-neutral text-neutral-content">
                 <div className="items-center daisy-card-body">
                   <h2 className="daisy-card-title">Balancer</h2>
-                  <img
+                  <Image
                     alt="balancer logo"
                     height={80}
                     width={80}
-                    src="https://cryptologos.cc/logos/balancer-bal-logo.svg?v=024"
+                    src="/images/balancer-bal-logo.svg"
                   />
                 </div>
               </div>
 
               <div className="w-48 daisy-card bg-neutral text-neutral-content">
                 <div className="items-center daisy-card-body">
                   <h2 className="daisy-card-title">Synthetix</h2>
-                  <img
+                  <Image
                     alt="snx logo"
                     height={80}
                     width={80}
-                    src="https://cryptologos.cc/logos/synthetix-network-token-snx-logo.svg?v=024"
+                    src="/images/synthetix-network-token-snx-logo.svg"
                   />
```

Comment:
Using the `Image` component is probably not worth bringing these files into the repo. We can just leave these as `img` with external `src` urls.

Reply:
Understood!